tests/performance/scale/tests/run-perf.sh

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env bash
```

Comment:
I guess this should be removed. It's probably starting a script refactored into multiple.

I'm using one simple method to keep such files in working copy but not in git. I have set global `gitignore`. Mostly to ignore IDE or OS-specific files. But I also have a record `mt_nogit_*`, that allows me to create dirs/files that will be persisted locally but not tracked.